nipype/pipeline/engine/utils.py

```diff
@@ -1055,7 +1055,8 @@ def export_graph(graph_in, base_dir=None, show=False, use_execgraph=False,
 def format_dot(dotfilename, format='png'):
     """Dump a directed graph (Linux only; install via `brew` on OSX)"""
     if format != 'dot':
-        cmd = 'dot -T%s -O \'%s\'' % (format, dotfilename)
+        formatted_dotfilename = os.path.splitext(dotfilename)[0]+'.'+format
+        cmd = 'dot -T{} -o"{}" "{}"'.format(format, formatted_dotfilename, dotfilename)
```

Comment:
why not:
```Python
cmd = 'dot -T{0} -o"{1}.{0}" "{2}"'.format(format, os.path.splitext(dotfilename)[0], dotfilename)
```

and avoid the concatenation?

Reply:
You need to add the indexes of the argument for format to use the first argument twice without doubling it explicitly. (ref. Chris' comment below)